app/app.go

```diff
@@ -471,23 +471,26 @@ func NewStrideApp(
 		keys[recordsmoduletypes.StoreKey],
 		keys[recordsmoduletypes.MemStoreKey],
 		app.GetSubspace(recordsmoduletypes.ModuleName),
-		app.IBCKeeper.ChannelKeeper,
-		&app.IBCKeeper.PortKeeper,
 		scopedRecordsKeeper,
 	)
 	recordsModule := recordsmodule.NewAppModule(appCodec, app.RecordsKeeper, app.AccountKeeper, app.BankKeeper)
 
+	// create IBC stacks by combining middleware with base application
+	// recordsStack contains records -> transfer
+	recordsStack := recordsmodule.NewIBCModule(app.RecordsKeeper, transferIBCModule)
```

Comment:
makes sense based on https://ibc.cosmos.network/main/ibc/middleware/integration.html

Reply:
we don't need to pass in a third arg because we are only setting a single scopedKeeper, right? per https://ibc.cosmos.network/main/ibc/middleware/integration.html#:~:text=customIBCModule1%20%3A%3D%20custom.NewIBCModule(customKeeper%2C%20scopedKeeperCustom1%2C%20%22portCustom1%22)

Reply:
I think you can optionally pass in a default port for the IBCModule to use, but we're just using "Transfer" in this case